### ISSUE TITLE
Authentication failed when cloning public repositories

### ISSUE THREAD

**Issue by kafku** [takezoe/gitbucket#717]
Hello,

I'm using gitbucket 3.1.1 running on [docker container](https://registry.hub.docker.com/u/f99aq8ove/gitbucket/) with the following configuration.
```
gravatar=true
notification=true
is_create_repository_option_public=true
ldap_authentication=false
ssh=false
smtp.host=smtp.mandrillapp.com
allow_account_registration=false
smtp.from_name=Gitbucket Notification
smtp.port=587
allow_anonymous_access=false
smtp.from_address=******
smtp.password=******
smtp.user=******
```

When I tried to clone my public repository on gitbucket,
authentication failed and I couldn't clone my repository.
```bash
$git clone http://my_gitbucket_server/git/fukui/test_public.git
Cloning into 'test_public'...
Username for 'http://my_gitbucket_server': fukui	
Password for 'http://fukui@my_gitbucket_server': 
fatal: Authentication failed for 'http://my_gitbucket_server/git/fukui/test_public.git/'
```
In contrast, cloning private repositories succeeded with the same configuration.
  
Thank you.

**Comment by samtron1412** [comment]
Same issue here.
Still log in successfully on web base, but when clone it fatal error authentication failed for public repositories.

**Comment by AllTaken** [comment]
I have encountered the same problem.

**Comment by takezoe** [comment]
I expect this issue has been fixed in #697.
Please wait for the next release at this weekend.
